packages/shared/src/lib/integrations.ts

```diff
@@ -0,0 +1,38 @@
+import { FunctionComponent } from 'react';
+import { IconProps } from '../components/Icon';
+import { AppIcon, SlackIcon } from '../components/icons';
+import { UserIntegrationType } from '../graphql/integrations';
+import { PromptOptions } from '../hooks/usePrompt';
+import { labels } from './labels';
+import { ButtonColor } from '../components/buttons/Button';
+
+const integrationTypeToIconMap: Record<
+  UserIntegrationType,
+  FunctionComponent<IconProps>
+> = {
+  [UserIntegrationType.Slack]: SlackIcon,
+};
+
+export const getIconForIntegration = (
+  type: UserIntegrationType,
+): FunctionComponent<IconProps> => {
+  return integrationTypeToIconMap[type] || AppIcon;
+};
+
+export const deleteSourceIntegrationPromptOptions: PromptOptions = {
+  title: labels.integrations.prompt.deleteSourceIntegration.title,
```

Comment:
Are these actually re-used or only in this file? (the labels)

Reply:
for now no, but was easier to just push them to lib because they could easily be

Reply:
Yeah no big feelings about it but wondering if it was one spot injection or not.